redirections/talks/is-my-robot-butler-going-to-steal-my-job.yml

```diff
@@ -0,0 +1 @@
+new_url: talks/a-p45-from-c3po/
```

Comment:
This needs to be preceded with a slash, i.e.

```yaml
new_url: /talks/a-p45-from-c3po/
```

otherwise it tries to redirect to `/talks/is-my-robot-butler-going-to-steal-my-job/talks/a-p45-from-c3po/`.